packages/manager/modules/pci/src/projects/project/storages/databases/components/order-review/order-review.html

```diff
@@ -0,0 +1,56 @@
+<h5 data-translate="pci_database_order_review_title"></h5>
+
+<dl>
+    <div class="row mb-2">
+        <dt
+            class="col-sm-3 font-weight-normal"
+            data-translate="pci_database_order_review_selected_database"
+        ></dt>
+        <dd class="col-sm-9 my-auto">
+            <span data-ng-bind="$ctrl.capitalize($ctrl.engine)"></span>
+            <span data-ng-bind="$ctrl.version"></span>
+        </dd>
+    </div>
+    <div class="row mb-2">
+        <dt
+            class="col-sm-3 font-weight-normal"
+            data-translate="pci_database_order_review_selected_plan"
+        ></dt>
+        <dd
+            class="col-sm-9 my-auto"
+            data-ng-bind="$ctrl.capitalize($ctrl.plan)"
+        ></dd>
+    </div>
+    <div class="row mb-2">
+        <dt
+            class="col-sm-3 font-weight-normal"
+            data-translate="pci_database_order_review_selected_flavor"
+        ></dt>
+        <dd
+            class="col-sm-9 my-auto"
+            data-ng-bind="$ctrl.capitalize($ctrl.flavor.name)"
+        ></dd>
+    </div>
+    <div class="row mb-2" data-ng-if="$ctrl.flavor">
+        <dt
+            class="col-sm-3 font-weight-normal"
+            data-translate="pci_database_order_review_price"
+        ></dt>
+        <dd class="col-sm-9 my-auto">
+            <ovh-manager-catalog-price
+                data-price="$ctrl.flavor.monthlyPrice.priceInUcents"
+                data-tax="$ctrl.flavor.monthlyPrice.tax"
+                data-user="$ctrl.user"
+            >
+            </ovh-manager-catalog-price>
+            <span data-translate="pci_database_common_price_monthly"></span>
+            (<ovh-manager-catalog-price
```

Comment:
Component is just displaying text ?

Reply:
yeah... created a component just to avoid cluttering of code